gradle/quality.gradle

```diff
@@ -50,11 +50,11 @@ configurations
 
 spotless {
    java {
-      importOrder(['static java', 'static javax', 'static org', 'static com', 'static scala', 'java', 'javax', 'org', 'com', 'scala'])
+      importOrder 'static java', 'static javax', 'static org', 'static com', 'static scala', 'java', 'javax', 'org', 'com', 'scala'
       // JDK 11 Hack: https://github.com/junit-team/junit5/commit/f39ba8d1e7b479bfd2f137f3b85cdeeb91ded22c
-      if (org.gradle.api.JavaVersion.current() == org.gradle.api.JavaVersion.VERSION_1_8) {
+      //if (org.gradle.api.JavaVersion.current() == org.gradle.api.JavaVersion.VERSION_1_8) {
```

Comment:
can we just remove this commented out code?

Reply:
oh yes!
good catch